README.md

```diff
@@ -5,6 +5,50 @@ Tools for building Pyodide.
 See [http://github.com/pyodide/pyodide](http://github.com/pyodide/pyodide) for
 more information.
 
+## Quickstart
+
+### Usage
+
+> [!TIP]
+> Currently, installing `pyodide-build` is preferred to running it.
+
+#### Install it
+
+```bash
+pip install pyodide-build
+pyodide --help
+pyodide build --help
```

Comment:
Why do you include these?

Reply:
I can probably drop it, it's just meant to serve as a filler as we don't have docs hosted yet. But a CLI is usually self-documenting, so it's fine not to include this.

Reply:
Addressed in b7708ff5707fda9d41d59cceb2bad1e52577f3fd.